.github/changes-filter.yaml

```diff
@@ -8,6 +8,7 @@ docs:
   - '.asf.yaml'
   - '*.md'
   - '**/*.md'
+  - '**/*.yaml'
```

Comment:
this rule would also match `.github/workflows/*.yaml` is that intentional? the problem is that workflow changes wouldn't be tested.

Reply:
it's just temporary to verify my pull